designdocs/capacity_staking_rewards_implementation.md

```diff
@@ -0,0 +1,324 @@
+# Capacity Staking Rewards Implementation
+The proposed feature is a design for staking FRQCY token in exchange for Capacity and/or FRQCY.
+It is specific to the Frequency Substrate parachain.
+It consists of enhancements to the capacity pallet, needed traits and their implementations, and needed runtime configuration.
+
+This does _not_ outline the economic model for Staking Rewards (also known as "Provider Boosting"); it describes the economic model as a black box, i.e. an interface.
+
+## Context and Scope:
+The Frequency Transaction Payment system uses Capacity to pay for certain transactions on chain.  Accounts that wish to pay with Capacity must:
+
+1. Have an [MSA](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/accounts.md)
+2. Be a [Provider](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/provider_registration.md) (see also [Provider Permissions and Grants](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/provider_permissions.md))
+3. Stake a minimum amount of FRQCY (on mainnet, UNIT on Rococo testnet) token to receive [Capacity](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/capacity.md).
+
+# Problem Statement
+This document outlines how to implement the Staking for Rewards feature described in [Capacity Staking Rewards Economic Model (TBD)](TBD), without, at this time, regard to what the economic model actually is.
```

Comment:
`, without, at this time, regard to what the economic model actually is.`  statement is a bit weird.